src/elements.jl

```diff
@@ -186,11 +201,12 @@ end
 
 
 struct ElementGaussian <: Element
-    α                               # Prefactor
-    L                               # Width of the Gaussian nucleus
+    α               # Prefactor
+    L               # Width of the Gaussian nucleus
     symbol::Symbol  # Element symbol
 end
 AtomsBase.atomic_symbol(el::ElementGaussian) = el.symbol
+AtomsBase.atomic_mass(::ElementGaussian) = nothing  # forces user to chose a mass
```

Comment:
You had this for the symbol as well before and it caused more issues than it solved. Let's leave it like it is, but we might need to change this to "zero mass" instead in the future.